manage.py

```diff
@@ -5,20 +5,8 @@
 
 
 def main():
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'src.settings')
-
-    # start new section
-    from django.conf import settings
-
-    if settings.DEBUG:
-        if os.environ.get('RUN_MAIN') or os.environ.get('WERKZEUG_RUN_MAIN'):
-            import ptvsd
-
-            ptvsd.enable_attach(address=('0.0.0.0', 3000))
-            ptvsd.wait_for_attach()
-            print('Attached!')
```

Comment:
@rscronce this is probably why debug wasn't working for you.

Reply:
Was a little more involved. Gunicorn has issues with ptvsd. We resolved it by reverting the dockerfile CMD to the original runserver command AND adding this code back.